assets/js/system-status.ts

```diff
@@ -7,65 +7,54 @@ window.addEventListener('DOMContentLoaded', () => {
   );
 
   const $operationalIcons = document.querySelectorAll('.operational');
-  const $degradedPerformanceIcons = document.querySelectorAll(
-    '.degraded-performance'
-  );
-  const $partialServiceDisruptionIcons = document.querySelectorAll(
-    '.partial-service-disruption'
-  );
-  const $plannedMaintenanceIcons = document.querySelectorAll(
-    '.planned-maintenance'
-  );
-  const $serviceDisruptionIcons = document.querySelectorAll(
-    '.service-disruption'
-  );
-  const $securityEventIcons = document.querySelectorAll('.security-event');
+  const $warningIcons = document.querySelectorAll('.degraded-performance');
```

Comment:
I think `warning` would be a more appropriate class name. 

`icon-status-degraded-performance.svg`  would also need to be updated with the new class name and a new file name such as `icon-status-warning.svg`